library/private/options_tools.h.in

```diff
@@ -125,8 +157,23 @@ double parse(const std::string& str)
 template<>
 ratio_t parse(const std::string& str)
 {
-  // TODO implement proper ratio parsing
-  return options_tools::parse<double>(str);
+  try
+  {
+    if (!str.empty() && str.at(str.size() - 1) == '%')
+    {
+      return parse_double(str.substr(0, str.size() - 1)) / 100;
+    }
+    return parse_double(str);
```

Comment:
supports fraction ? 

eg: `1/3` ?

Reply:
slippery slope to full blown calculator but if we limit to fractions i could see it

Reply:
I think it's useful too. Much easier to set `1/30` when setting the framerate than `0.0333333333333333`.